docs/client.md

```diff
@@ -0,0 +1,53 @@
+# pb-cli specification
+
+## Overview
+
+`pb-cli` is a command line tool which communicate with `pb-go` backend. Although user can just upload data to pastebin directly with `curl`, we still provide a tools to help user easily use our advanced feature such as burn-after-reading and private share.
+
+We guarantee everything that worked with `pb-cli`, also worked with `curl`.
+
+## Configuration
+
+You can use `-c file` to provide a config file. If you didn't provide one, it will find config file called `.pb-cli.toml` from current directory then home directory.
+
+Config file should be written in `toml` like:
+
+```toml
+host = "http://your.pastebin.bakcend:port"
+master_key = "Same as your config on server"
```

Comment:
The masterkey can be not exists since not all users are administrator of a site.

Reply:
Suggestion: `default_expire` can be used here since it can reduce user input.